worldedit-core/src/main/java/com/sk89q/worldedit/function/mask/OffsetMask.java

```diff
@@ -30,7 +30,10 @@
 /**
  * Checks whether another mask tests true for a position that is offset
  * a given vector.
+ *
+ * @deprecated Use {@link OffsetsMask#single}
  */
+@Deprecated
```

Comment:
Is it forRemoval as well?

Reply:
That's a question for @me4502 to answer. Hopefully, she can clarify for us.

Reply:
Most likely, generally deprecations are for removal in the next major version

there’s not much point having two classes that do the same thing here